subsys/bluetooth/audio/Kconfig.tbs

```diff
@@ -170,6 +170,158 @@ config BT_TBS_CLIENT_MAX_TBS_INSTANCES
+	bool "Read Bearer URI Schemes Supported List Procedure"
+	default y
+	help
+	  This option enables support for the Read Bearer URI Schemes Supported
+	  List procedure.
+
+config BT_TBS_CLIENT_BEARER_SIGNAL_STRENGTH
+	bool "Read Bearer Signal Strength Procedure"
+	default y
+	help
+	  This option enables support for the Read Bearer Signal Strength
+	  procedure.
+
+config BT_TBS_CLIENT_READ_BEARER_SIGNAL_INTERVAL
+	bool "Read Bearer Signal Strength Reporting Interval Procedure"
+	default y
+	help
+	  This option enables support for the Read Bearer Signal Strength
+	  Reporting Interval procedure.
+
+config BT_TBS_CLIENT_SET_BEARER_SIGNAL_INTERVAL
+	bool "Set Bearer Signal Strength Reporting Interval Procedure"
+	default y
+	help
+	  This option enables support for the Set Bearer Signal Strength
+	  Reporting Interval procedure.
+
+config BT_TBS_CLIENT_BEARER_LIST_CURRENT_CALLS
+	bool "Read Bearer List Current Calls Procedure"
+	default y
+	help
+	  This option enables support for the Read Bearer List Current Calls
+	  procedure.
+
+config BT_TBS_CLIENT_CCID
+	bool "Read Content Control ID Procedure"
+	default y
+	help
+	  This option enables support for the Read Content Control ID
+	  procedure.
+
+config BT_TBS_CLIENT_INCOMING_URI
+	bool "Read Incoming Call Target Bearer URI Procedure"
+	default y
+	help
+	  This option enables support for the Read Incoming Call Target Bearer
+	  URI procedure.
+
+config BT_TBS_CLIENT_STATUS_FLAGS
+	bool "Read Status Flags Procedure"
+	default y
+	help
+	  This option enables support for the Read Status Flags Procedure
+	  procedure.
+
+config BT_TBS_CLIENT_CP_PROCEDURES
+	bool
+	default y
+	depends on (BT_TBS_CLIENT_CP_ACCEPT_CALL) || \
+	BT_TBS_CLIENT_CP_TERMINATE_CALL || \
+	BT_TBS_CLIENT_CP_HOLD_CALL || \
+	BT_TBS_CLIENT_CP_RETRIEVE_CALL || \
+	BT_TBS_CLIENT_CP_ORIGINATE_CALL || \
+	BT_TBS_CLIENT_CP_JOIN_CALLS
+	help
+	  This hidden option indicates that there are at least one control
+	  point procedure available.
+	  When this option is disabled it indicates that the control point can
+	  be optimized away.
+
+config BT_TBS_CLIENT_CP_ACCEPT_CALL
+	bool "Call Control Point Answer Incoming Call Procedure"
+	default y
+	help
+	  This option enables support for the Call Control Point Answer
+	  Incoming Call procedure.
+
+config BT_TBS_CLIENT_CP_TERMINATE_CALL
+	bool "Call Control Point Terminate Incoming Call Procedure"
+	default y
+	help
+	  This option enables support for the Call Control Point Terminate
+	  Incoming Call procedure.
+
+config BT_TBS_CLIENT_CP_HOLD_CALL
+	bool "Call Control Point Move Call To Local Hold Procedure"
+	default y
+	help
+	  This option enables support for the Call Control Point Move Call To
+	  Local Hold procedure.
+
+config BT_TBS_CLIENT_CP_RETRIEVE_CALL
+	bool "Call Control Point Move Locally Held Call To Active Call Procedure"
+	default y
+	help
+	  This option enables support for the Call Control Point Move Locally
+	  Held Call To Active Call procedure.
```

Comment:
General comment: Since we do not expose any of these as "Control point operations" in the API, we should perhaps consider removing the "CP" part of the names and omit the "control point" part of the descriptions